anvio/dbops.py

```diff
@@ -3421,7 +3421,7 @@ def __init__(self, db_path, run=run, progress=progress, quiet=True):
 
 
     def init(self):
-        if os.path.exists(self.db_path):
+        if filesnpaths.is_file_exists(self.db_path):
```

Comment:
@meren I tried your suggestion in a previous comment use `self.pan_db = PanDatabase(self.pan_db_path)` to verify the PanDatabase path exists but when I tested it with a fake path no `ConfigError` was raised. Could this be a solution?

Reply:
This would've been even better:

```
utils.is_pan_db(self.db_path)
```